Assets/Scripts/UI/LevelSelectionHandle.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+
+public class LevelSelectionHandle : MonoBehaviour
+{
+    [SerializeField]
+    List<int> _disabledButtonsIndexFromZero = new List<int>();
+    [SerializeField]
+    List<int> _hideButtonsIndexFromZero = new List<int>();
+
+    private void Start()
+    {
+        DisableButtonsFromList();
+        HideButtonsFromList();
+    }
+
+    private void DisableButtonsFromList()
+    {
+        var buttons = GetComponentsInChildren<Button>();
+        for (int i = 0; i < buttons.Length; i++)
+        {
+            if (_disabledButtonsIndexFromZero.Contains(i))
+            {
+                buttons[i].interactable = false;
+            }
+        }
+    }
+
+    private void HideButtonsFromList()
+    {
+        var buttons = GetComponentsInChildren<Button>();
+        for (int i = 0; i < buttons.Length; i++)
+        {
+            if (_hideButtonsIndexFromZero.Contains(i))
+            {
+                buttons[i].gameObject.SetActive(false);
+            }
+        }
```

Comment:
Having trouble figuring out why we need these as separate lists and separate functions, generally IIRC when an object is SetActive(false), it shouldn't be interactable either way. And if you do need both of them separately, is there a point to have two separate functions for this, seems like you could do both in one for.